javascript-data-model/order-history.js

```diff
@@ -0,0 +1,30 @@
+var orderHistory = [
+  {
+    orderPlaced: 'August 4, 2020',
+    total: '$34',
+    product: 'JavaScript for impatient programmers',
+    author: 'Rauschmayer, Dr.Axel',
+    shipTo: 'JS Masher'
+  },
+  {
+    orderPlaced: 'July 19, 2020',
+    total: '$44.53',
+    product: 'The Timeless Way of Building',
+    author: 'Alexander Christopher',
+    shipTo: 'JS Masher'
+  },
+  {
+    orderPlaced: 'July 4, 2020',
+    total: '$17.22',
+    product: 'Gamecube Contoller Adapter, Super Smash Bros Switch Gamecube Adapter for  WII U, PC Support Turbo and Vibration Features. No Driver and No Lag- Gamecube Adapter',
+    shipTo: 'JS Masher'
+  },
+  {
+    orderPlaced: 'July 3, 2020',
+    total: '$138.93',
+    product: 'GameCube Controller- Super Smash Bros Edition (Nintendo Switch)',
+    product2: 'The Art of Sql',
```

Comment:
Instead of separate product properties, try making an array of objects to represent each product in an order. The other orders each have one product, but it should still be an array with one object in it.